plugins/test/unit/plugins/distributors/yum/test_publish.py

```diff
@@ -155,7 +155,7 @@ def test_publish(self, mock_publish_distribution, mock_publish_rpms, mock_publis
         mock_publish_rpms.assert_called_once()
         mock_publish_drpms.assert_called_once_with(mock_publish_distribution(),
                                                    repo_content_unit_q=None)
-        mock_publish_errata.assert_called_once_with(repo_content_unit_q=None)
+        mock_publish_errata.assert_called_once()
```

Comment:
All export tests are currently broken/skipped. I'll try to fix them. 
This fixed test above is for publish not for the export.